docker/README.md

```diff
@@ -11,4 +11,33 @@ For base image source code and documentation please refer to https://github.com/
 
 To build install s2i build tool from: https://github.com/openshift/source-to-image/releases/tag/v1.1.3 
 
```

Comment:
@wtrocki - Maybe describe that this was the initial step used to create the aerogear image (i.e docker.io/aerogear/jenkins-1-centos7) and is currently used as the base image now - wdyt ?